package.json

```diff
@@ -141,7 +141,7 @@
     "whatwg-fetch": "^3.6.2"
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=20.0.0"
```

Comment:
So. My thinking here is that we pin both of the dev environments (Docker and Conda) to the same specific version (22.8.0), but we relax the constraint here. And to make things easier on ourselves, we drop official support for Node 18.

My understanding is that changing the value in package.json will not prevent somebody from running the app with Node 18; it will just warn them. However:

- when I ran `yarn install` after making this change, I did not get any warnings
- when I tried it with `npm install` (v8.19.2), I did get a warning (but just a warning, it didn't exit with an error):
  `npm WARN EBADENGINE Unsupported engine`

Reply:
My thinking behind pinning the dev versions to the same version is just to avoid the possibility of a node version related bug appearing in one environment but not the other and causing confusion

Reply:
My only hesitation about pinning on the conda environment, as listed before, is that with hard pins, one has to be quite disciplined about updating them regularly. Otherwise, one misses out on security updates and the like.

I know this is true of the Docker image too, but unless it is infra/core system deps or to avoid hard incompatibilities then it is best to not add a hard pin to `major.minor`